Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:18.17.0-alpine
```

Comment:
Don't you think it will too latest? can we downgrade to node 16

Reply:
Sure should node 16 be okay?

Reply:
Done node image set to v.16.20.2